tensorflow/core/framework/tensor.cc

```diff
@@ -703,14 +703,14 @@ void PrintOneDim(int dim_index, gtl::InlinedVector<int64, 4> shape, int64 limit,
 }
 
 template <typename T>
-string SummarizeArray(int64 limit, int64 num_elts, const TensorShape &tensor_shape, 
```

Comment:
Sorry, it's fine to use a const ref, just collate the ampersand to the type, not the variable name.